packages/server/src/logger.ts

```diff
@@ -72,3 +72,20 @@ function getCloudWatchLogger(config: MedplumServerConfig): CloudWatchLogger {
   }
   return cloudWatchLogger;
 }
+
+export function parseLogLevel(level: string): LogLevel {
```

Comment:
You can use the enum to do this rather than duplicating values:

```
function parseLogLevel(level: string): LogLevel {
  const value = LogLevel[level as keyof typeof LogLevel];
  
  if (value === undefined) {
    throw new Error(`Invalid log level: ${level}`);
  }
  
  return value;
}
```